main.go

```diff
@@ -108,6 +108,10 @@ func main() {
 							Usage:  "Optional. Specify whether SAM should skip pulling down the latest Docker image. Default is false.",
 							EnvVar: "SAM_SKIP_PULL_IMAGE",
 						},
+						cli.StringFlag{
```

Comment:
Notes for future: common options like profile must be abstracted into an `args` object and passed down to Invoke/StartApi for processing.